jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/BottomSheetTests.java

```diff
@@ -0,0 +1,50 @@
+package io.github.epam.angular.tests.elements.complex;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.bottomSheet;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class BottomSheetTests extends TestsInit {
+    private static final List<String> BOTTOM_SHEET_VALUES = Arrays.asList("Google Keep", "Google Docs", "Google Plus", "Google Hangouts");
+
+    @BeforeClass(alwaysRun = true)
+    public void before() {
+        shouldBeLoggedIn();
+        angularPage.shouldBeOpened();
+        bottomSheet.show();
+    }
+
+    @Test
+    public void checkBottomSheetIsDisplayedTest() {
+        bottomSheet.open();
+        bottomSheet.is().opened();
+    }
+
+    @Test
+    public void checkBottomSheetIsHiddenTest() {
+        bottomSheet.open();
+        bottomSheet.is().opened();
```

Comment:
This check is not required: bottomSheet.is().opened();

Reply:
Fixed, thanks.

Reply:
Maybe this is some kind of misunderstanding. Could you please delete this check.

Reply:
Got you, fixed.